js/init.js

```diff
@@ -1 +1,51 @@
-//Select branch please
\ No newline at end of file
+const API = 'https://jsonplaceholder.typicode.com/users/';
+let promiseArray = [];
+let dataFromApi = [];
+
+
+//XHR
+for (let i = 1; i <= 10; i++) {
+    promiseArray[i] = new Promise((resolve, reject) => {
+       const xhr = new XMLHttpRequest();
+       xhr.open("GET", API + i);
+       xhr.onload = function() {
+           if (this.status === 200) {
+               resolve(JSON.parse(xhr.response));
+           } else {
+               reject({
+                   status: this.status,
+                   statusText: xhr.statusText
+               });
+           }
+       };
+       xhr.onerror = function() {
+            xhr.responseText;
+       };
+       xhr.send();
+    });
+}
+
+Promise.all(promiseArray).then(
+    data => data.forEach(function (el, index) {
```

Comment:
```suggestion
{
    data => dataFromApi = data;
    console.log(dataFromApi);
}
```

результат, который вы, вроде бы, хотите получить:)